src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { Post } from '../../types/Post';
+
+import './PostList.scss';
+
+type Props = {
+  posts: Post[]
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo key={post.id} post={post} />
```

Comment:
Consider using a bunch of props here for a better understanding of what is needed for your info component directly